app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -85,13 +85,15 @@ public function getAdditionalData(array $excludeAttr = [])
 
                 if (!$product->hasData($attribute->getAttributeCode())) {
                     $value = __('N/A');
```

Comment:
I think we also should remove this code as there are a lot of complaints about such behavior. This code was "dead" for a long time so everyone expects that not used attribute is ignored on storefront.
Please remove this `if`

Reply:
Can we change system behavior in a way desired by Community without PO approval?

Reply:
Actually, behavior was changed when `instanceof Phrase` was added to `if` below. I can see that `__('N/A')` code was dead for a long time (including M1) so expectations in most Magento versions not used attributes are not displayed on a storefornt product page. Support also has multiple tickets to return previous behavior (not show N/A) back.

Reply:
Thanks, I wan't aware about M1, and yes, such useful behavior was introduced accidentally in scope of another bugfix.